### ISSUE TITLE
Signature field not saving value

### ISSUE THREAD

**Issue by bigskinnyboy** [verbb/formie#923]
**Description**
Signature field displays canvas but doesn't pass a value back on submit. If this is marked as a required field, you get a validation error, otherwise blank value. I've only tried this on Chrome MacOS so far.

I'm not sure if an image is supposed to be rendered in the DOM or whether it is attached as a data uri for the field value. There is no image outputted and it looks like no value set for the hidden input.

**Steps to reproduce**
1. Add signature field to form and mark required
2. Draw on canvas, submit form
3. Input value is not passed in payload

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: dev-craft-4 2.0.0-beta.10
- Craft version: Craft Solo 4.0.3
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Completely missed the breaking change from [signature_pad](https://github.com/szimek/signature_pad).

Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by bigskinnyboy** [comment]
Thank you.

Confirmed, latest update works for me. 🥳

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.11](https://github.com/verbb/formie/releases/tag/2.0.0-beta.11)
